governance.md

```diff
@@ -1,34 +1,61 @@
+The Kyma project also includes the [Kyma Incubator](https://github.com/kyma-incubator) organization. It is a place where all new projects start in a more relaxed environment that facilitates their rapid growth. At that stage, they do not have to comply with all the rules that govern the Kyma organization. Once the incubating project is ready to become a part of the main Kyma organization, adjust it to all the standards.
 
-## Maintainers' responsibilities
+## Ownership policy
 
-With great power comes great responsibility. Maintainers not only make changes and approve pull requests but also truly care about their projects.
+Kyma repositories are owned by code owners who are a group of people with special privileges in the repositories of the [GitHub](../../../) organization. Each repository has a separate `CODEOWNERS` file located at its root. The file specifies persons who have the ability to approve contribution to the part of the repository they own. after a detailed review of the related pull requests. Although the name suggests only the code ownership, the `CODEOWNERS` file is not only about the code but the content in general. Apart from the developers, you can define any relevant parties as code owners. For example, technical writers are set up as the owners of all `.md` documents in the Kyma repositories and SIG/WG members are the owners of their SIG/WG's folders' content.
 
-Every maintainer is expected to:
+### Code owners' responsibilities
 
-* contribute high-quality code and content
-* communicate and collaborate with other maintainers to improve the maintenance process
-* perform a thorough review of incoming pull requests
-* approve only those pull requests in which the contributor made the requested improvements
-* check if the related CI job has completed successfully before approving the pull request
-* make sure that the pull request approval flow runs smoothly
-* prioritize issues and manage their processing
-* proactively fix bugs
-* perform maintenance tasks for their projects
+With great power comes great responsibility. Code owners not only review and approve pull requests but also truly care about their projects.
 
-## Become a maintainer
+Every code owner is expected to:
 
-To become a maintainer, contact one of the existing maintainers of the given project and agree to fulfill the requirements listed above.
-The existing maintainer can also propose a new member by creating a pull request adding the candidate to the `CODEOWNERS` file in the project's repository. The change requires the two-third majority of all maintainers' votes.
+* Contribute high-quality code and content
+* Communicate and collaborate with other code owners to improve the ownership process
+* Perform a thorough review of incoming pull requests and make sure it follows the contributing rules from the [`CONTRIBUTING.md`](CONTRIBUTING.md) document.
```

Comment:
`Perform a thorough review of incoming pull requests and make sure they follow the contributing rules`...